ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -399,6 +399,9 @@ private static void createLines(UcteNetworkExt ucteNetwork, Network network) {
         for (UcteLine ucteLine : ucteNetwork.getLines()) {
             UcteNodeCode nodeCode1 = ucteLine.getId().getNodeCode1();
             UcteNodeCode nodeCode2 = ucteLine.getId().getNodeCode2();
+            if (!nodeCode1.getVoltageLevelCode().equals(nodeCode2.getVoltageLevelCode())) {
+                throw new UcteException("Line with two different nominal voltages");
+            }
```

Comment:
I think this test should be done in the `UcteElement::fix` method.
@murgeyseb what do you think?
@jlabous You should give more information in the error message: with your proposal, if this exception is thrown, I don't know which line is the faulty one so it's impossible to debug/fix it.

Reply:
Yes it could be done in UcteElement::fix, it is true. And the message should be more explicit.
But this method is supposed to fix the issue, not throw exception, but why not.

Reply:
So maybe we can do the test in the constructor then?